docs/gpu-feature-discovery/README.md

```diff
@@ -210,6 +210,8 @@ their meaning:
 | nvidia.com/gpu.memory          | Integer    | Memory of the GPU in Mb                                                                                                                                                                | 2048           |
 | nvidia.com/gpu.product         | String     | Model of the GPU                                                                                                                                                                       | GeForce-GT-710 |
 | nvidia.com/gpu.mode            | String     | Display or Compute Mode of the GPU. Details of the GPU modes can be found [here](https://docs.nvidia.com/grid/13.0/grid-gpumodeswitch-user-guide/index.html#compute-and-graphics-mode) | compute        |
+| nvidia.com/gpu.clique          | String     | GPUFabric ClusterUUID + CliqueID                                                                                                                               | 7b968a6d-c8aa-45e1-9e07-e1e51be99c31-1 |
```

Comment:
Can labels contain `.`? Do we want to use `.` as a separator instead so that we can parse these if required?

Reply:
you mean the value? so we will have ` 7b968a6d-c8aa-45e1-9e07-e1e51be99c31.1` ?

Reply:
to your question, yes dots are allowed 
https://github.com/kubernetes/apimachinery/blob/master/pkg/util/validation/validation.go#L31

Reply:
I think something more obvious like `:` or `/` might be more appropriate. Even `::`

Reply:
I don't feel strongly though. I do agree that it probably shouldn't be `-` though since that is already in the UUID.

Reply:
yeah that regex only allows `-` or `.`

Reply:
I think `.` makes sense, making it easier to parse, given that the hash contains `-`

Reply:
I set it to `.`,  hope we all agree is the best for parsing